storage/range_tree_test.go

```diff
@@ -31,8 +31,9 @@ import (
 // SetupRangeTree is called via store.BootstrapRange.
 func TestSetupRangeTree(t *testing.T) {
```

Comment:
You should look at the `biogo.store/llrb` tests for inspiration. I hope you looked at `biogo.store/llrb` for inspiration on the code was well.